WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -357,14 +357,17 @@ public async Task<Coin> OutpointToCoinAsync(InputRegistrationRequest request, Ca
 		return new Coin(input, txOutResponse.TxOut);
 	}
 
-	public async Task<RoundState[]> GetStatusAsync(RoundStateRequest request, CancellationToken cancellationToken)
+	public async Task<RoundStateResponse> GetStatusAsync(RoundStateRequest request, CancellationToken cancellationToken)
 	{
 		using (await AsyncLock.LockAsync(cancellationToken).ConfigureAwait(false))
 		{
-			return Rounds.Select(x => {
+			var roundStates = Rounds.Select(x =>
+			{
 				var checkPoint = request.RoundCheckpoints.FirstOrDefault(y => y.RoundId == x.Id);
 				return RoundState.FromRound(x, checkPoint == default ? 0 : checkPoint.StateId);
 			}).ToArray();
+
+			return new RoundStateResponse(roundStates, Array.Empty<CoinJoinFeeRateAvarage>());
```

Comment:
I needed to deal with the common interface - the idea comes from @lontivero. What would be your solution?

Reply:
Can you clarify it a bit? What do you mean with "the common interface"? What must be common?

Reply:
IWabiSabiApiRequestHandler

Reply:
I have been looking into this and at the moment, I don't have a better solution because anything else would lead to some refactorings you probably want to avoid at this point. 

btw: I feel that `WabiSabiHttpApiClient` can be improved but we can do it later too.